com.trollworks.gcs/src/com/trollworks/gcs/utility/Log.java

```diff
@@ -30,6 +31,17 @@
             } catch (Throwable throwable) {
                 error("Unable to redirect log to " + property, throwable);
             }
+        } else if (Platform.isWindows()) {
+            String localAppData = System.getenv("LOCALAPPDATA");
+            if (localAppData != null) {
+                File logDirectory = new File(localAppData);
```

Comment:
You dropped the `+ "\\GCS"` part, which should either still be there, or the `+ "\\error.log"` cases below should be revised to something like `+ "\\gcs_error.log"`.

Reply:
You have the patience of a saint, thank you. I applied an theme to IntelliJ and it tries to mark what code does inline, I'm gonna go turn that off so I don't make that mistake again.